keyboards/cannonkeys/link/rules.mk

```diff
@@ -0,0 +1,2 @@
+SERIAL_DRIVER = vendor
```

Comment:
This should be migrated to `keyboard.json` under `split.serial.driver`.

Reply:
Done, thank you